src/content/ja/updates/2019/02/rendering-on-the-web.md

```diff
@@ -42,16 +37,11 @@ book_path: /web/updates/_book.yaml
 
 *サーバーレンダリングはナビゲーションに応じたサーバー上のページのために、完全なHTMLを生成します。これはブラウザが応答を受ける前に処理されるため、データフェッチとクライアント上の成形による追加のラウンドトリップを回避します。*
 
-サーバーレンダリングは一般的に、迅速な [First Paint](/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint) (FP) と [First
-Contentful Paint](/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint) (FCP) を生み出します。
-ページのロジックとレンダリングをサーバー上で実行すると、Javascriptをクライアントへ大量送信することが回避できます。これは短時間の [Time to Interactive](/web/tools/lighthouse/audits/time-to-interactive) (TTI) を達成するのを助けます。サーバーレンダリングは、テキストとリンクをユーザーのブラウザへ送信するだけなので理にかなっています。
-このアプローチは、広範囲のデバイスやネットワークの状態に適しており、ストリーミングドキュメントのパースのように興味深いブラウザの最適化が見られます。
+サーバーレンダリングは通常、高速の[ファーストペイント](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint) （FP）および[ファーストコンテンツフルペイント](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint) （FCP）を生成します。サーバー上でページロジックを実行してレンダリングすることにより、クライアントに大量のJavaScriptを送信することを回避できるため、高速な[対話型](https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive) （TTI）を実現できます。サーバーレンダリングでは、テキストとリンクをユーザーのブラウザーに送信するだけなので、これは理にかなっています。このアプローチは、デバイスおよびネットワークの状態の広い範囲でうまく機能し、ストリーミングドキュメント解析などの興味深いブラウザ最適化を開きます。
```

Comment:
Parsed referencing links. What can we do about these?

Reply:
@svasilenkov Looks like GitLocalize auto-parse referenced links and inline them. Do we have an option to turn it off? I think it's a feasible glitch but would like to keep original and translation in sync. Also, will this properly work if original is partially updated? Won't the links be lost?